src/Migrations/Version20230315102027.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230315102027 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE centre DROP FOREIGN KEY FK_C6A0EA756885AC1B');
+        $this->addSql('ALTER TABLE gestionnaire DROP FOREIGN KEY FK_F4461B20A76ED395');
```

Comment:
maybee the 2 migrations are squashable ?

Reply:
No, we need to execute the first one, use the gestionnaire migration command (and disable associations command), then execute the second migration. It is the reason why I initially opened 2 different PR.